src/main/com/example010/radek/v2/Soldier.java

```diff
@@ -0,0 +1,60 @@
+package main.com.example010.radek.v2;
+
+public class Soldier implements BaseFighter {
+
+    private String name;
+    private int life;
+
+    public int attack() {
+
+        return 0;
+    }
+
+    public void attack(Soldier soldier) {

```

Comment:
zadnou logiku nebudeme drzet v domain objektech ... nechme tu jen jejich atributy a settry, gettery, nic vic!

Reply:
Ok, přišlo mi to čistější, když takto stačila jedna logika, která je stejná pro oba bojovníky, dokud se nebudou nějak rozlišovat

Reply:
cistejsi to je presne naopak ,... logiku resi service ... v nasem pripade Battle ... BattleService